server/src/main/java/de/uftos/entities/Subject.java

```diff
@@ -33,6 +33,8 @@ public class Subject {
   @NotEmpty
   private String name;
 
+  private String color;
```

Comment:
You should also update the the `SubjectRequestDto`

Reply:
Oh true, sorry